vcd/resource_vcd_nsxt_edgegateway.go

```diff
@@ -63,35 +63,50 @@ func resourceVcdNsxtEdgeGateway() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
-			"org": &schema.Schema{
+			"org": {
 				Type:     schema.TypeString,
 				Optional: true,
 				ForceNew: true,
 				Description: "The name of organization to use, optional if defined at provider " +
 					"level. Useful when connected as sysadmin working across different organizations",
 			},
-			"vdc": &schema.Schema{
-				Type:        schema.TypeString,
-				Optional:    true,
-				ForceNew:    true,
-				Description: "The name of VDC to use, optional if defined at provider level",
+			"vdc": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Computed:      true,
+				Description:   "The name of VDC to use, optional if defined at provider level",
+				ConflictsWith: []string{"owner_id", "starting_vdc_id"},
+				Deprecated:    "This field is deprecated in favor of 'owner_id' which supports both - VDC and VDC group IDs",
+			},
+			"owner_id": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Computed:      true,
+				Description:   "ID of VDC Group or VDC",
```

Comment:
Thinking about the new users here. Small usability suggestion for all the descriptions of this field: let's put VDC first (and Group second), as the most popular use case will become VDC ID as soon as we drop the deprecated "vdc" field.

E.g.:
"VDC or VDC Group ID"

Reply:
Changed in here and data source.